3.preprocessing_features/scripts/3.aggregate_single_cells.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Aggregating single cells
+# In this notebook, we process single-cell feature-selected profiles to generate compound-level aggregated profiled for each plate using the pycytominer. The single-cell profiles are grouped by treatment (Metadata_treatment) and are saved as Parquet files in the aggregated_profiles directory. These aggregated profiles provide concise and interpretable data for downstream analysis at the compound level.
```

Comment:
I would like to keep the steps consistent with other projects. Just to get this correct, the order is single -> aggregate -> normalize -> fs? (this would be a separate PR) I think I'll do this after my mAP implementation PR. 

Also, I feel this way is highly important for my metric because it takes into account single cells, and imbalances of cell counts could affect the results. We can also use this as an opportunity to see how much that cell count really affects my metric as well depending on how we process the features "Cameron's method" vs "other methods"